common/djangoapps/xblock_django/management/commands/list_plugin_modules.py

```diff
@@ -0,0 +1,12 @@
+"""
+List plugin modules.
+"""
+
+from django.core.management.base import BaseCommand
+from ...plugin_translations import get_xblock_module_names
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for xblock_module_name in get_xblock_module_names():
```

Comment:
TODO: XBlock entry point name differs in Drag and Drop XBlock:

```
'xblock.v1': 'drag-and-drop-v2 = drag_and_drop_v2:DragAndDropBlock'
```

Which one should we use.